src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -692,9 +768,20 @@ protected bool IsAlive
                 return !ElementIDLifecycleManager<int>.GetInstance().IsRevitDeleted(InternalElementId.IntegerValue);
             }
         }
+        /// <summary>
+        /// Finds the elements that are joined with the given element.
+        /// </summary>
+        /// <returns>All elements joined to the given element</returns>
+        public Element[] GetJoinedElements()
```

Comment:
In general it is best to use interface types when defining apis - IEnumerable<Element> rather than array unless there is a reason to use array.

Reply:
Yep, agree - didn't check for this as most of the nodes in that same category are returning arrays. @SHKnudsen fixed this now though.